__tests__/components/Button.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Button } from '../../src/components';
+
+describe('Button', () => {
+  const onPress = { navigate: jest.fn() };
+
+  it('renders a default button', () => {
+    const tree = renderer.create(<Button title={'title'} onPress={onPress} />);
```

Comment:
the Button component can have the optional prop `invert`. we should have a second test with that case as well.

Reply:
done in 9d3b4be5b92af2f9b3b9669be38ae70ad29230e1